corporate/tests/test_stripe.py

```diff
@@ -1670,6 +1671,18 @@ def test_approve_sponsorship(self) -> None:
         self.assertEqual(message.recipient.type, Recipient.PERSONAL)
         self.assertEqual(message.recipient_id, recipient_id)
 
+    def test_update_sponsorship_status(self) -> None:
+        lear = get_realm("lear")
+        iago = self.example_user("iago")
+        update_sponsorship_status(lear, True, acting_user=iago)
+        self.assertTrue(Customer.objects.filter(realm=lear).first().sponsorship_pending)
```

Comment:
@hackerkid in the review you suggested to use `get_customer_by_realm` function. But, that has a problem, it's return type is `Optional [UserProfile]` and when I did `get_customer_by_realm(realm).sponsorship` mypy is warning against it... so I have kept the change as it was before... is this fine???